src/components/NewWorkspaceModal.js

```diff
@@ -33,21 +33,27 @@ const styles = {
   label: {
     ...Style.elements.sectionHeader,
     marginTop: '1rem', marginBottom: '0.25rem'
+  },
+  groupNotice: {
+    marginBottom: '0.25rem',
+    fontSize: 12,
+    color: Style.colors.textFaded
   }
 }
 
 export default class NewWorkspaceModal extends Component {
   constructor(props) {
     super(props)
+    const { cloneWorkspace } = props
     this.state = {
       billingProjects: undefined,
       allGroups: undefined,
-      name: '',
-      namespace: undefined,
-      description: '',
-      groups: [],
+      name: cloneWorkspace ? `Copy of ${cloneWorkspace.workspace.name}` : '',
```

Comment:
Checked with @bradtaylor, UX is better as `Oldname copy` rather than `Copy of oldname` due to alphabetical sorting position

Reply:
Ah, good!